config/config.go

```diff
@@ -26,6 +26,34 @@ var (
 		Name:      "config_last_reload_success_timestamp_seconds",
 		Help:      "Timestamp of the last successful configuration reload.",
 	})
+
+	// DefaultModule set default configuration for the Module
+	DefaultModule = Module{
+		HTTP: DefaultHTTPProbe,
+		TCP:  DefaultTCPProbe,
+		ICMP: DefaultICMPProbe,
+		DNS:  DefaultDNSProbe,
+	}
+
+	// DefaultHTTPProbe set default value for HTTPProbe
```

Comment:
Fullstop

Reply:
It's needed at config.go:186, and if a remove it, it will not load the default config (tested). 
Maybe I'm missing something, just let me known.